### DISCUSSION TITLE
make some parameters on logs-panel configurable via variables

### DISCUSSION THREAD

**Discussion by yarix** [grafana/grafana#38317] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
the ability to configure "wrap lines" (and maybe other params) in Logs panel via dashboard variable.
![image](https://user-images.githubusercontent.com/6409857/123625180-2bc3d080-d818-11eb-94d5-79ab832269bc.png)

**Why is this needed**:
as a viewer, i would like to be able to control the line-wrapping of the logs on the screen.
this option is availble now only for editors, on the panel properties.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
